src/util/util.ts

```diff
@@ -58,18 +62,8 @@ export function checkPropsUndefined(props: object): boolean {
   return someUndefined;
 }
 
-// see https://stackoverflow.com/a/38552302
 export function parseJwt(token: string): TokenPayload {
-  const base64Url = token.split(".")[1];
-  const base64 = base64Url.replace(/-/g, "+").replace(/_/g, "/");
-  const jsonPayload = decodeURIComponent(
-    atob(base64)
-      .split("")
-      .map((c) => "%" + ("00" + c.charCodeAt(0).toString(16)).slice(-2))
-      .join("")
-  );
-
-  return JSON.parse(jsonPayload);
+  return JSON.parse(atob(token.split(".")[1]));
```

Comment:
Might be nice to move this into a named variable first so it's easier to know what `[1]` is.